src/utils/validate_account.cairo

```diff
@@ -7,11 +7,13 @@ use starknet::ContractAddress;
 /// Validates an account.
 /// # Arguments
 /// * `account` - Account to validate.
-fn validate_account(account: ContractAddress) { // TODO
+fn validate_account(account: ContractAddress) {
+    assert(account.is_non_zero(), 'satoru/null-account');
```

Comment:
error message in a constant in an error file, following conventions of other error messages

Reply:
thanks didn't catch that